src/modules/GroupChannel/components/MessageInputWrapper/index.tsx

```diff
@@ -11,10 +14,28 @@ export interface MessageInputWrapperProps {
 }
 
 export const MessageInputWrapper = (props: MessageInputWrapperProps) => {
+  const { config } = useSendbirdStateContext();
   const context = useGroupChannelContext();
+  const {
+    messages,
+    currentChannel,
+  } = context;
+  const isLastMessageSuggestedRepliesEnabled = useIIFE(() => {
+    if (!config?.groupChannel?.enableSuggestedReplies) return false;
+    if (getSuggestedReplies(lastMessage).length === 0) return false;
+    const lastMessageInContext = messages[messages.length - 1];
+    if (isSendableMessage(lastMessageInContext) && lastMessageInContext.sendingStatus !== 'succeeded') return false;
+
+    return true;
+  });
+  const lastMessage = currentChannel?.lastMessage;
```

Comment:
```suggestion
  const lastMessage = currentChannel?.lastMessage;
  const isLastMessageSuggestedRepliesEnabled = useIIFE(() => {
    if (!config?.groupChannel?.enableSuggestedReplies) return false;
    if (getSuggestedReplies(lastMessage).length === 0) return false;
    const lastMessageInContext = messages[messages.length - 1];
    if (isSendableMessage(lastMessageInContext) && lastMessageInContext.sendingStatus !== 'succeeded') return false;

    return true;
  });
```

Reply:
Can you move the declaration of `lastMessage` upwards? It's being used in `getSuggestedReplies`